core/libp2p/protocol/kademlia/query_runner.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_KAD_QUERY_RUNNER_HPP
+#define KAGOME_KAD_QUERY_RUNNER_HPP
+
+#include "libp2p/protocol/kademlia/common.hpp"
+#include "libp2p/protocol/kademlia/query.hpp"
+
+namespace libp2p::protocol::kademlia {
+
+struct QueryRunner {
```

Comment:
Docs please

Reply:
I do not understand this myself, will add after it is implemented